### ISSUE TITLE
Elements should have sufficient color contrast on background color

### ISSUE THREAD

**Issue by akslay** [az-digital/arizona-bootstrap#309]
## Problem/Motivation
Color contrast is built-in for text on a colored background, but list item bullets and links are not.

### Describe the bug

- You can't see list item bullets on top of a darker colored background.
- Links remain red on top of background colors
- Heading color contrast was previously working, but has stopped working (need to investigate if another change impacted this)

### To Reproduce

- Place a heading element on a dark background color
- Place a bulleted/numbered list on top of a dark background color (i.e., AZ Blue).
- Place a link on top of a colored background

## Proposed resolution

- Add list item pseudo elements to the SCSS rule that ensures sufficient color contrast
- Add links to the SCSS rule that ensures sufficient color contrast

### Expected behavior

- List item pseudo elements should be the same color as the text when placed on a background color
- Links should change colors when not sufficient color contrast

##  Screenshots
<img width="1181" alt="color_contrast" src="https://user-images.githubusercontent.com/52931751/122272981-cd9d0200-ce95-11eb-9a29-56beeb94e6ca.png">

**Comment by akslay** [comment]
It looks like regular list items (ol/ul) properly inherit the text color. But lists using the utility class: `.ul-triangles` do not. So that will be the only list item that needs fixing.
